eng/pipelines/client.yml

```diff
@@ -1,20 +1,12 @@
 # DO NOT EDIT THIS FILE
 # This file is generated automatically and any changes will be lost.
 
-pr:
-  branches:
-    include:
-    - master
-  paths:
-    include: # Changes in these directories cause us to build everything for validation purpoes.
-    - eng/
-    - .azure-pipelines/
 
-trigger:
-- master
-
-variables:
-  ProjectFile: eng/data-plane.proj
+# I'm disabling this build for now since I don't think we need it.
+trigger: none
+pr: none
 
 jobs:
-- template: templates/jobs/archetype-sdk-client.yml
\ No newline at end of file
+- template: templates/jobs/archetype-sdk-client.yml
+  parameters:
+    ProjectFile: eng/data-plane.proj
```

Comment:
This will conflict with my PR https://github.com/Azure/azure-sdk-for-net/pull/6321. Can you change it to 
```
ProjectFile: $(ProjectFile)
```

Reply:
Done.